ffi/ffi-macros/src/ontology.rs

```diff
@@ -141,6 +142,21 @@ impl From<Slot> for CSlot {
     }
 }
 
+impl AsRust<Slot> for CSlot {
+    fn as_rust(&self) -> Fallible<Slot> {
+        Ok(
+            Slot {
+                raw_value: create_rust_string_from!(self.raw_value),
+                value: self.value.as_rust()?,
+                range: (self.range_start as usize..self.range_end as usize),
+                entity: create_rust_string_from!(self.entity),
+                slot_name: create_rust_string_from!(self.slot_name),
+                confidence_score: Some(self.confidence_score),
```

Comment:
It should rather be:
```rust
confidence_score: if self.confidence_score < 0 { None } else { Some(self.confidence_score) } 
```

Reply:
Oh yesss, thanks for catching this one 🙊 !